src/input.rs

```diff
@@ -30,17 +30,11 @@ fn setup(mut commands: Commands) {
 #[derive(Clone, Copy)]
 pub struct MidiSettings {
     pub is_debug: bool,
```

Comment:
The reason for note_on and off is due to some keyboards using other indices than 144 and 128. See #7 
I'm not sure if the `NOTE_ON_SIGNATURE` and `NOTE_OFF_SIGNATURE` bitmask works for every midi keyboard.

Reply:
Oh yeah I should have explained this change.

All note off events have status `0b1000xxxx`, where the `x`s are the channel of the message (e.g. `0b10000000` for channel 1, `0b10001111` for channel 16). Same thing for note on events having the status `0b1001xxxx`. See [here](https://www.music.mcgill.ca/~ich/classes/mumt306/StandardMIDIfileformat.html#BMA1_1). 

I'm pretty confident that this fix I made would fix #7, because 156 and 140 are `0b10011100` and `0b10001100`, which are note on and off events in channel 13, and these new methods would correctly identify them as note on/off events.